variables.tf

```
@@ -11,6 +11,11 @@ variable "talos" {
       api_server         = string
       kubernetes_version = string
       talos_version      = string
+      extra_mounts       = optional(map(object({
+        path    = string
+        type    = string
+        options = list(string)
```

```suggestion
        options = optional(list(string))
```
Options should be optional as might not need any options when doing a mount